### PR TITLE
Update social demo schema to use valid social metadata

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Updates the `social-demo` schema to use valid social metadata claims as per the new [v2 launch pattern](https://github.com/ONSdigital/ons-schema-definitions/blob/v3/docs/rm_to_eq_runner_payload_v2.rst)

Adds a demo "Health" themed schema

Needed to test new launch payload structure for runner.

### CODE REVIEW

File: schemas/social/en/social_demo.json [hunk 1]
```diff
@@ -748,21 +748,21 @@
...
     },
     "metadata": [
         {
-            "name": "user_id",
+            "name": "case_ref",
             "type": "string"
         },
         {
-            "name": "period_id",
+            "name": "case_type",
```
Comment (MebinAbraham) [inline schemas/social/en/social_demo.json:761]: This can be dropped.

File: schemas/social/en/social_demo.json [hunk 2]
```diff
@@ -748,21 +748,17 @@
...
+    "theme": "social",
     "navigation": {
         "visible": false
     },
     "metadata": [
         {
-            "name": "user_id",
+            "name": "case_ref",
             "type": "string"
         },
```
Comment (MebinAbraham) [inline schemas/social/en/social_demo.json:759]: Should be removed.

File: schemas/health/en/health_demo.json [hunk 3]
```diff
@@ -0,0 +1,765 @@
...
+                            "skip_conditions": [
+                                {
+                                    "when": [
+                                        {
+                                            "id": "answer6dcf7f90-6dac-48f6-8bc3-5e1f2fc77f22",
+                                            "condition": "equals any",
+                                            "values": [
+                                                "No"
+                                            ]
+                                        }
```
Comment (MebinAbraham) [inline schemas/health/en/health_demo.json:707]: Can you ensure these are all updated to use new rules format.

File: schemas/social/en/social_demo.json [hunk 4]
```diff
@@ -410,19 +410,17 @@
                                     }
```
Comment (MebinAbraham) [inline schemas/social/en/social_demo.json:410]: The survey_id needs to be changed for this also

**Review by MebinAbraham — APPROVED** [review]

**Review by Yuyuutsu — APPROVED** [review]

**Pull request merged**